model/appender_test.go

```diff
@@ -920,7 +921,8 @@ func validateFile(t *testing.T, fileName string) {
 
 	handler, _ := sighandler.NewAdobeX509RSASHA1(nil, nil)
 	handler2, _ := sighandler.NewAdobePKCS7Detached(nil, nil)
-	handlers := []model.SignatureHandler{handler, handler2}
+	handler3, _ := sighandler.NewDocTimeStamp("", 0)
+	handlers := []model.SignatureHandler{handler, handler2, handler3}
```

Comment:
Can we get any information about the timestamp here?  Like at what time it was applied?
Would be good to add to the test case to print out some timestamp info.
Is there anything that would make sense to add to SignatureValidationResult  ?
For example if printing out signature info such as with example
https://github.com/unidoc/unipdf-examples/blob/master/signatures/pdf_sign_validate.go

Reply:
I'll create an example to https://github.com/unidoc/unipdf-examples

Reply:
Great

Reply:
I added an example to my fork https://github.com/a5i/unipdf-examples/blob/timestamp_sign/signatures/pdf_sign_validate_timestamp.go
I'm waiting to merge the PR to change the version of Unipdf in the examples.